src/info/langs/language.rs

```diff
@@ -104,7 +109,7 @@ impl std::fmt::Display for LanguagesInfo {
 fn prepare_languages(
     languages_info: &LanguagesInfo,
     color_palette: &[DynColors],
-) -> Vec<(String, f64, DynColors)> {
+) -> Vec<(String, f64, DynColors, &'static str)> {
```

Comment:
This tuple is getting kind of long, and IMO hard to read. It might be time to use a struct.

Reply:
I replaced the tuple with a struct.
I wasn't sure about the name, but `LanguageDisplayData` was the best I could come up with.